lib/order.rb

```diff
@@ -0,0 +1,92 @@
+require_relative "customer.rb"
+require "csv"
+
+# Build Order Class and instantiate instance variables include readers for all attributes
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status
+  VALID_STATUSES = [:pending, :paid, :processing, :shipped, :complete]
+
+  # Build initialize method to take in parameters with a default fulfillment status.
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+
+    # Validate that input was among the included possible statuses. Raise error, otherwise.
+    if !VALID_STATUSES.include?(fulfillment_status)
+      raise ArgumentError.new("Invalid Status")
```

Comment:
It's helpful for debugging to say what status was invalid.

```suggestion
      raise ArgumentError.new("Invalid Status: #{fulfillment_status}")
```